x-pack/plugins/security_solution/cypress/README.md

```diff
@@ -115,8 +115,42 @@ cd x-pack/plugins/security_solution
 CYPRESS_BASE_URL=http(s)://<username>:<password>@<kbnUrl> CYPRESS_ELASTICSEARCH_URL=http(s)://<username>:<password>@<elsUrl> CYPRESS_ELASTICSEARCH_USERNAME=<username> CYPRESS_ELASTICSEARCH_PASSWORD=password yarn cypress:run:firefox
 ```
 
+#### CCS Custom Target + Headless
+
+This test execution requires two clusters configured for CCS. See [Search across clusters](https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-cross-cluster-search.html) for instructions on how to prepare such setup.
+
+The instructions below assume:
+* Search cluster is on server1
+* Remote cluster is on server2
+* Remote cluster is accessible from the search cluster with name `remote`
+* Security and TLS are enabled
+
+```shell
+# bootstrap Kibana from the project root
+yarn kbn bootstrap
+
+# launch the Cypress test runner with overridden environment variables
+cd x-pack/plugins/security_solution
+CYPRESS_ELASTICSEARCH_USERNAME="user" \
+CYPRESS_ELASTICSEARCH_PASSWORD="pass" \
+CYPRESS_BASE_URL="https://user:pass@server1:5601" \
+CYPRESS_ELASTICSEARCH_URL="https://user:pass@server1:9200" \
+CYPRESS_CCS_KIBANA_URL="https://user:pass@server2:5601" \
+CYPRESS_CCS_ELASTICSEARCH_URL="https://user:pass@server2:9200" \
+CYPRESS_CCS_REMOTE_NAME="remote" \
+yarn cypress:run:ccs
+```
+
+Similar sequence, just ending with `yarn cypress:open:ccs`, can be used for interactive test running via Cypress UI.
+
+Appending `--browser firefox` to the `yarn cypress:run:ccs` command above will run the tests on Firefox instead of Chrome.
```

Comment:
Just to double-check, is `ccs` a test run mode for running _all_ and _any_ Cypress tests, or only a subset of tests? I guess it only runs tests in the `ccs_integration` folder, but want to make sure this is a correct understanding.

Also, does the normal test run exclude `ccs_integration` folder?

In other words, are "normal" and "ccs" test runs _completely_ separate in terms of tests and configs?

Reply:
@banderror the CCS mode executes all the tested inside the `ccs_integration` folder and the normal execution would execute that the tests inside the `integration` folder. So yes, are completely separate runs in terms test, configs and also when those tests are going to be executed.

We have separated the tests  because what are we going to check on CCS is not exactly the same as what we check on a regular scenario as well as the data, the data is going to differ.